public class frequencyCheck{
 public static void main(String[]args){
        int[] arr={3,4,5,6,7,3};
        for(int i=0;i<arr.length;i++){
            int count=0;
            for(int j=0;j<arr.length;j++){
                if(arr[i]==arr[j]){
                    count++;
                }
            }
            System.out.println(arr[i] +"is there"+ count +" "+ "times");
        }
    }
